### PR TITLE
Remove done items from TODO

### PR DESCRIPTION
**Pull request by JackPlowman**

### CODE REVIEW

**Review by stefaniuk — DISMISSED** [review]
👍

File: TODO.md [hunk 1]
```diff
@@ -2,20 +2,15 @@
 
 ## Priority list
 
-- Application
-  - PUT vs. PATCH
 - Documentation
   - Generate API Documentation automatically (CSAPI-9)
 - Testing
-  - Check for unit and integration test completeness (CSAPI-64)
   - Can unit and integration tests be run independently
```
Comment (Hrdy26) [inline TODO.md:8]: Can we clarify that this is running the tests independently from the db

File: TODO.md [hunk 2]
```diff
@@ -2,20 +2,15 @@
...
   - Implement smoke test suite
 
 ## Other tasks
 
 - Security
-  - API throttling (CSAPI-44)
-  - Toggle Django debug mode using environment variable
   - Use the library Docker NGINX image for proxy and make sure it is in the latest version
 - Application
   - Let's revisit the domain model and consider again the naming of `serviceUid` (just `id`), `serviceName` (just `name`?), `capacityStatus` (just `status`?)
```
Comment (Hrdy26) [inline TODO.md:16]: Think we can remove this

File: TODO.md [hunk 3]
```diff
@@ -2,20 +2,15 @@
 
 ## Priority list
```
Comment (Hrdy26) [inline TODO.md:4]: Can we add the splitting the migration to it's own stage in the pipeline as a priority task please

**Review by Hrdy26 — DISMISSED** [review]

**Review by Hrdy26 — APPROVED** [review]

**Pull request merged**